definition/rest/v1/settings.ts

```diff
@@ -10,38 +10,38 @@ type SettingsUpdatePropsActions = {
 export type OauthCustomConfiguration = {
 	_id: string;
 	clientId?: string;
-	custom: unknown;
+	custom: boolean;
 	service?: string;
-	serverURL: unknown;
-	tokenPath: unknown;
-	identityPath: unknown;
-	authorizePath: unknown;
-	scope: unknown;
+	serverURL: string;
+	tokenPath: string;
+	identityPath: string;
+	authorizePath: string;
+	scope: string;
 	loginStyle: 'popup' | 'redirect';
-	tokenSentVia: unknown;
-	identityTokenSentVia: unknown;
-	keyField: unknown;
-	usernameField: unknown;
-	emailField: unknown;
-	nameField: unknown;
-	avatarField: unknown;
-	rolesClaim: unknown;
-	groupsClaim: unknown;
-	mapChannels: unknown;
-	channelsMap: unknown;
-	channelsAdmin: unknown;
-	mergeUsers: unknown;
-	mergeRoles: unknown;
-	accessTokenParam: unknown;
-	showButton: unknown;
-
-	appId: unknown;
+	tokenSentVia: string | boolean;
+	identityTokenSentVia: string | boolean;
+	keyField: string | boolean;
+	usernameField: string | boolean;
+	emailField: string | boolean;
+	nameField: string | boolean;
+	avatarField: string | boolean;
```

Comment:
I didn't find any place using these keys as `boolean`, only as `string` .. I'd like to understand why you added `boolean` for them :)